package.json

```
@@ -3,12 +3,13 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "NODE_ENV=test npx knex migrate:latest --env test",
```

```suggestion
    "tests:migration": "NODE_ENV=test npx knex migrate:latest --env test",
```

We are going to add unit tests in future. Let's use the `tests:migration` command.